packages/quantic/force-app/main/default/lwc/quanticSort/quanticSort.js

```diff
@@ -75,11 +81,22 @@ export default class QuanticSort extends LightningElement {
   schemas = {};
 
   labels = {
-    sortBy,
     relevancy,
     newest,
     oldest,
     invalidCustomSortConfig,
+    sortBy,
+  };
+
+  variants = {
+    default: {
+      name: 'default',
+      label: sortBy,
+    },
+    wide: {
+      name: 'wide',
+      label: undefined,
```

Comment:
Even the wide variant has a label, which is the word `sort by displayed` above the wide sort by input.
I don't think it's a good idea to remove the label as it creates an inconsistency as in with one variant the label is included and with another variant the label is not and the variant being just related to the size of the input.

so I think it would be better to make the variants work in the following way:

-  variant default: display the label and a small sort input beside it.
-  variant wide: display the label and a wide sort input below it.

this way out component stays consistent and scalable as if in the future we need a sort component without a label we can expose a new property label that can be modified without impacting the size of the sort input.